addons/marketing_card/static/src/scss/mass_mailing.scss

```diff
@@ -0,0 +1,12 @@
+body.o_mass_mailing_iframe_body_fullscreen {
+    display: grid;
+    align-items: center;
+}
+
+body.o_in_iframe {
+    background: $gray-200;
+}
```

Comment:
:warning: This selector is also wildly generic and impact more than just your usecase.